src/serlo-editor/plugins/sc-mc-exercise/renderer.tsx

```diff
@@ -77,7 +77,7 @@ export function ScMcExerciseRenderer({
                   />
                   <label
                     className={clsx(
-                      'serlo-spacing-fix flex cursor-pointer items-center [&>div>*]:ml-2',
+                      'serlo-spacing-fix flex cursor-pointer items-center [&_.slate-container]:mb-0 [&_.slate-p]:ml-2',
```

Comment:
Do you want to have this here or should we try to fix it in `serlo-spacing-fix` where it worked before?

Reply:
in `serlo-tailwind.css`

Reply:
Ah, I see that you have added `.serlo-spacing-fix` specifically for this place.

Gut feeling: using specific fixes like in this PR is slightly more understandable.

Todo: also update mc exercise